docs/reference/components/sinks/datadog.cue

```diff
@@ -36,13 +36,27 @@ components: sinks: _datadog: {
 			}
 		}
 		endpoint: {
-			common:      false
-			description: "The endpoint to send logs to."
-			required:    false
+			common:        false
+			description:   "The endpoint to send data to."
+			relevant_when: "region is not set"
+			required:      false
 			type: string: {
-				default: "intake.logs.datadoghq.com:10516"
+				default: null
 				examples: ["127.0.0.1:8080", "example.com:12345"]
 			}
 		}
+		region: {
+			description:   "The region to send data to."
+			required:      false
+			relevant_when: "endpoint is not set"
+			warnings: []
+			type: string: {
+				enum: {
+					us: "United States"
+					eu: "Europe"
+				}
+				examples: [ "us"]
```

Comment:
Mind rebasing from `master` and removing this line? `examples` are no longer required when `enum` is provided since we can derive the examples.

Reply:
Will do.